src/asset/svg/EyeIcon.tsx

```diff
@@ -0,0 +1,41 @@
+interface EyeIcon {
+  visible?: boolean;
+  handleEyeIconClick?: () => void;
```

Comment:
아이콘은 어짜피 아이콘 하나라는 의미가 있기때문에 무엇을 클릭한다라기보다, 클릭한다라는 의미만 표현되어도 상관없을 것 같아요~
```suggestion
  onClick?: () => void;
```